app/models/cart_item.rb

```diff
@@ -14,4 +16,18 @@ def unit_price
   def total_price
     unit_price * quantity
   end
+
+  private
+
+  def enough_items_present_in_pricing
```

Comment:
Assignment Branch Condition size for enough_items_present_in_pricing is too high. [17.29/15]